src/acceptance/assets/app/go_app/deploy.sh

```diff
@@ -1,37 +1,44 @@
 #!/bin/bash
-
 set -euo pipefail
-config=${CONFIG:-}
+
+config="$(cat "${CONFIG:-}")"
+
 function getConfItem(){
-  val=$(jq -r ".$1" "${config}")
+  val=$(jq -r ".$1" <<< "${config}")
   if [ "$val" = "null" ]; then return 1; fi
   echo "$val"
 }
+
 if [ -z "${config}" ]; then
   echo "ERROR: Please supply the config using CONFIG env variable"
   exit 1
 fi
 
-script_dir=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
-app_dir="$( realpath -e "${script_dir}/build")"
-service_name=$(getConfItem service_name)
+script_dir="$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &> /dev/null && pwd)"
+
+app_dir="$(realpath -e "${script_dir}/build")"
+app_domain="$(getConfItem apps_domain)"
+service_name="$(getConfItem service_name)"
+memory_mb="$(getConfItem node_memory_limit||echo 128)"
+service_broker="$(getConfItem service_broker)"
+service_plan="$(getConfItem service_plan)"
```

Comment:
Yeah, reading all necessary values before `pushd` is ececuted makes the fix in https://github.com/cloudfoundry/app-autoscaler-release/pull/2658/files#r1498839264 irrelevant but let's keep the fix since less error prone that way